### DISCUSSION TITLE
play a saved program on tc420

### DISCUSSION THREAD

**Discussion by moeff** [wallneradam/tc420#1] [Q&A]
big thx for this, making it possible to control my tc420 aquarium led controller by iobroker / alexa etc 👍 

but i have problems with the usage of the commands.

its possible to play a custom mode, saved on the controller? and also its possible to set it, without a time counting = unlimited time? 

thanks for any help.

**Comment by wallneradam** [comment]
No, it is not possible to read anything from controller. It is a hardware limitation, the device is write only.
But you can create shell or python scripts with all the parameters for play mode then you can modify them any time. The original pled.exe do the same, just saves parameters locally.

Also note that play mode is a little bit different, because it is not wall clock based. When you start it, it will always start at first timestep. And you need to give it the time period instead a time. 
I'm planning to make another version of play command, which can eat the same parameters as mode, then you can specify the start time to play from and the speed multiplier. Though I had no time for this yet.
